frontend/packages/console-shared/src/components/editor/SidebarLink.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+interface SidebarLinkProps {
+  onToggleSidebar?: () => {};
+}
+
+const SidebarLink: React.FC<SidebarLinkProps> = ({ onToggleSidebar }) => (
+  <>
+    <div className="co-action-divider">|</div>
+    <div className="ocs-yaml-editor__link">
```

Comment:
This should be controlled by the parent.

Reply:
Done.